assets/scripts/admin/adminNewArtists.jsx

```diff
@@ -1,140 +1,42 @@
-							<td className="col-xs-1">66</td>
-							<td className="col-xs-2">333</td>
-							<td className="col-xs-1">blahblahblah@balls.onu</td>
-						</tr>
-						<tr className="">
-							<td className="col-xs-2 pad-t-sm">
-								<img className="mobile-img-thumbnail" src="http://www.minxsociety.com/wp-content/uploads/2012/01/lola-blanc-music.jpg"></img>
-							</td>
-							<td className="col-xs-2">Lola Blanc</td>
-							<td className="col-xs-2">22</td>
-							<td className="col-xs-2">12</td>
-							<td className="col-xs-1">626</td>
-							<td className="col-xs-2">333</td>
-							<td className="col-xs-1">blahblahblah@balls.onu</td>
-						</tr>
-						<tr className="">
-							<td className="col-xs-2 pad-t-sm">
-								<img className="mobile-img-thumbnail" src="https://f4.bcbits.com/img/a3960757277_10.jpg"></img>
-							</td>
-							<td className="col-xs-2">Avarice</td>
-							<td className="col-xs-2">22</td>
-							<td className="col-xs-2">12</td>
-							<td className="col-xs-1">266</td>
-							<td className="col-xs-2">333</td>
-							<td className="col-xs-1">blahblahblah@balls.onu</td>
-						</tr>
-						<tr className="">
-							<td className="col-xs-2 pad-t-sm">
-								<img className="mobile-img-thumbnail" src="http://www.altpress.com/images/made/chrootimages/uploads/news/news_glassjaw_633_461_70_s_c1.jpg"></img>
-							</td>
-							<td className="col-xs-2">Glassjaw</td>
-							<td className="col-xs-2">22</td>
-							<td className="col-xs-2">12</td>
-							<td className="col-xs-1">222</td>
-							<td className="col-xs-2">333</td>
-							<td className="col-xs-1">blahblahblah@balls.onu</td>
-						</tr>
-					</tbody>
-				</table>
-			</div>
-		)
-	}
-});
+var AdminNewArtists = React.createClass({
+	getInitialState: function() {
+		return{
+			artist: []
 
-var AdminNewArtistsChart = React.createClass({
+		}
+	},
 	componentDidMount: function() {
-		var chart = c3.generate({
-			bindto: '.admin-chart',
-		  data: {
-		    json: [{
-		       day: 0,
-		      'Artists': 45,
-		    }, {
-		      day: 1,
-		      'Artists': 210,      
-		    }, {
-		      day: 2,
-		      'Artists': 100,
-		    }, {
-		      day: 3,
-		      'Artists': 400,
-		    }, {
-		    	day: 4,
-		      'Artists': 160,
-		    }, {
-		    	day: 5,
-		      'Artists': 250,
-		    }, {
-		    	day: 6,
-		      'Artists': 70,
-		    }, {
-		    	day: 7,
-		      'Artists': 140,
-		    }, {
-		    	day: 8,
-		      'Artists': 300,
-		    }],
-		    keys: {
-		      x: 'day',
-		      value: ['Artists']
-		    }
-		  },
-		  axis: {
-		    y: {
-		    	label: {
-		    		text:'# of users',
-		    		position: 'outer-middle'
-		    	}
-		    }
-		  }
-		});
-	},	
-
+		stemApi.getArtistSignups({
+    })
+    .then(function(response) {
+      this.setState({artist: response});
+      console.log(this.state.artists);
+    }.bind(this))
+    .catch(function(error) {
+      console.log('Creator Profile Error: ' + JSON.stringify(error));
```

Comment:
Change this console.log -> console.error, and use Utilities.normalizeError instead of JSON.stringify()